<launch>
    <arg name="chip" default="gpiochip0"/>
    <arg name="offset" default="4"/>
    <arg name="neopixel_rosserial" default="false" />
    <arg name="neopixel_python" default="true" />
    <arg name="output" default="log" />
    <arg name="ns" default="/piksi_multi_cpp_base/base_station_receiver_0"/>

    <include file="$(find piksi_multi_interface)/launch/interface.launch" pass_all_args="true"/>
</launch>
